### ISSUE TITLE
Uncaught TypeError: debouncePromise is not a function

### ISSUE THREAD

**Issue by jamesremuscat** [slorber/awesome-debounce-promise#9]
Trying to incorporate `awesome-debounce-promise` into my project, but whenever my debounced function is called I get:

```
Uncaught TypeError: debouncePromise is not a function
    at DebounceCache.getDebouncedFunction (index.es.js:222)
    at Library.AwesomeDebouncePromiseWrapper (index.es.js:248)
    at onChange (Library.js:247)
... (React internals below here)
```

From debugging it seems like `debouncePromise` is an object with the desired function as `default`, e.g. in `src/index.ts`

`import * as DebouncePromise from 'debounce-promise';`

might need to be

`import DebouncePromise from 'debounce-promise';`

unless I'm misunderstanding / doing something wrong?

**Comment by rpadilla6** [comment]
I have the same issue, here is my error code:
`Uncaught (in promise) TypeError: debouncePromise is not a function
    at DebounceCache.getDebouncedFunction (index.es.js:222)
    at AwesomeDebouncePromiseWrapper (index.es.js:248)
    at Object._callee$ (FullTextSearch.js:34)
    ***more disgusting stuff****
`

**Comment by slorber** [comment]
Hi,

Sorry about that and thanks for noticing this.
I'm not yet a TS expert but I think I fixed it. Tests were passing but it was due to different TS config... used default synthetic imports option and it seems the build works fine with the correct import now.

Published 2.1.0
